bb-aem-core/src/main/java/com/cognifide/qa/bb/aem/core/component/dialog/dialogfields/DialogField.java

```diff
@@ -49,4 +50,9 @@
   default String getLabel() {
     return null;
   }
+
```

Comment:
Better solution would be default here. Thanks to this it wouldn't required empty methods and would not break any API

Reply:
Also, document the new behaviour please :)

Reply:
https://media1.tenor.com/images/c5d41bfa655c636286d80ae4552ce386/tenor.gif?itemid=10337157

Reply:
Fixed & fixed